tests/test_flows.py

```diff
@@ -98,6 +98,24 @@ def get_bridge_mac(self):
             bridge = 'br-' + bridge_id[:12]
         return bridge
 
+    def __capture(self, test_index, test_name, bridge, duration=None):
+        
+        # `test_index` currently unused, but might be useful for file output
+        
+        # for optional file output
+        # self.outputfile = os.path.join(
+        #     self.rootdir, 'logs', f'tshark_log_{test_name}.json')
+        
+        try:
+            tshark_args = ['tshark', '-Q', '-i', bridge, '-T', 'json']
```

Comment:
If this is going to be used for actual hardware, not just for docker, there may be quite a lot of traffic that we're not interested in. Therefore, it would be good to add a pcap filter:
```suggestion
            tshark_args = ['tshark', '-Q', '-i', bridge, '-f', 'ether proto 0x88CC or ether proto 0x893A', '-T', 'json']
```